src/core/js/app.js

```diff
@@ -111,6 +113,18 @@ require([
         console.error(e);
     }
 
+    function configureInview() {
+        
+        var adaptConfig = Adapt.course.get("_inview");
```

Comment:
do we want this setting in course.json? seems to me like it would fit in config.json better